### DISCUSSION TITLE
Information on filter types

### DISCUSSION THREAD

**Discussion by CodingGreen** [TanStack/table#2292] [General]
On the documentation page about useFilter it says "For more information on filter types, see Filtering" (https://github.com/tannerlinsley/react-table/blame/master/docs/api/useFilters.md#L33)
But I can't seem to find the documentation it refers to and I can't find any documentation for the built-in filter types.

**Comment by dolie** [comment]
If someone coming by, the filter types are here : 
https://github.com/tannerlinsley/react-table/blob/master/src/filterTypes.js
